tex/pgfcircbipoles.tex

```diff
@@ -4720,6 +4720,126 @@
     \pgfusepath{draw}
 }
 
+% Opening normal closed Switch 
+\pgfcircdeclarebipolescaled{switches}
+{}
+{\ctikzvalof{bipoles/ncs/depth}}
+{oncs}
+{\ctikzvalof{bipoles/ncs/height}}
+{\ctikzvalof{bipoles/ncs/width}}
+{
+	
+	\pgf@circ@setlinewidth{bipoles}{\pgfstartlinewidth}
+	\pgfpathmoveto{\pgfpoint{\pgf@circ@res@left}{0pt}}
+	\pgfpathlineto{\pgfpoint{.9\pgf@circ@res@right}{\pgf@circ@res@up}}
+	\pgfusepath{draw}
+	\pgfsetlinewidth{\pgfstartlinewidth}
+	\pgfpathmoveto{\pgfpoint{\pgf@circ@res@right}{0pt}}
+	\pgfpathlineto{\pgfpoint{.2\pgf@circ@res@right}{0pt}}
+	\pgfpathlineto{\pgfpoint{.2\pgf@circ@res@right}{\pgf@circ@res@up}}
+	\pgfusepath{draw}
+	
+	\pgf@circ@setlinewidth{bipoles}{\pgfstartlinewidth}
+	\pgfscope
+	\pgftransformshift{\pgfpoint{\pgf@circ@res@left}{0pt}}
+	\pgfpathmoveto{\pgfpointpolar{-10}{.9\pgf@circ@res@right}}
+	\pgfpatharc{-10}{90}{.9\pgf@circ@res@right}
+	\pgfcirc@set@arrows{switch}{}{latexslim}
+	\pgfsetbeveljoin
+	\pgfusepath{draw}
+	\endpgfscope
+}
+
+%% Closing normal closed Switch
+\pgfcircdeclarebipolescaled{switches}
+{}
+{\ctikzvalof{bipoles/ncs/depth}}
+{cncs}
+{\ctikzvalof{bipoles/ncs/height}}
+{\ctikzvalof{bipoles/ncs/width}}
+{
+	
+	\pgf@circ@setlinewidth{bipoles}{\pgfstartlinewidth}
+	\pgfpathmoveto{\pgfpoint{\pgf@circ@res@left}{0pt}}
+	\pgfpathlineto{\pgfpoint{.9\pgf@circ@res@right}{\pgf@circ@res@up}}
+	\pgfusepath{draw}
+	\pgfsetlinewidth{\pgfstartlinewidth}
+	\pgfpathmoveto{\pgfpoint{\pgf@circ@res@right}{0pt}}
+	\pgfpathlineto{\pgfpoint{.2\pgf@circ@res@right}{0pt}}
+	\pgfpathlineto{\pgfpoint{.2\pgf@circ@res@right}{\pgf@circ@res@up}}
+	\pgfusepath{draw}
+	
+	\pgf@circ@setlinewidth{bipoles}{\pgfstartlinewidth}
```

Comment:
Please indent the code (4 spaces, when needed). I prefer to use the same thickness as the others switches, then we can add a configurable parameter like "modifier thickness" for resistors/etc.